web/src/popover_menus.js

```diff
@@ -307,13 +307,35 @@ export function initialize() {
                 return;
             }
 
-            $popper.one("click", ".sidebar-popover-mute-topic", () => {
-                muted_topics_ui.mute_topic(stream_id, topic_name);
+            $popper.one("click", ".sidebar-popover-toggle-unmute", () => {
+                const new_visibility_policy = user_topics.topic_has_visibility_policy(
+                    stream_id,
+                    topic_name,
+                    user_topics.visibility_policy.UNMUTED,
+                )
+                    ? user_topics.visibility_policy.INHERIT
+                    : user_topics.visibility_policy.UNMUTED;
```

Comment:
Long ternary clauses like this are really hard to read. Either use a variable for `is_unmuted`, an if/else block, or both, but don't do this :)